examples/mayavi/data_interaction/normal_flipping_stl.py

```diff
@@ -0,0 +1,201 @@
+"""
+This example shows how to read a stl file and flip it's normals.
+"""
+
+import sys
+from tvtk.api import tvtk
+from tvtk.common import configure_input
+import numpy as np
+from mayavi import mlab
+import tempfile
+
+cube_stl = """solid cube
```

Comment:
You could actually check this file into `examples/mayavi/data/cube.stl` or so and just use that.